<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1353</width>
    <height>785</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1021</width>
    <height>785</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1532</width>
    <height>785</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Egym demo1</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="g1">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>-2</y>
      <width>1536</width>
      <height>768</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(77, 77, 77);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="renderHints">
     <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::NonCosmeticDefaultPen|QPainter::Qt4CompatiblePainting|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-10</y>
      <width>381</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>54</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color: rgb(255, 255, 255) }</string>
    </property>
    <property name="text">
     <string>Ruderzug</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>643</x>
      <y>16</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Light</family>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color: rgb(87, 129, 170) }</string>
    </property>
    <property name="text">
     <string>Sandro</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>453</x>
      <y>664</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color: rgb(255, 255, 255) }</string>
    </property>
    <property name="text">
     <string>59</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="sek">
    <property name="geometry">
     <rect>
      <x>533</x>
      <y>686</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Light</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color: rgb(255, 128, 0) }</string>
    </property>
    <property name="text">
     <string>sek</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="training_title">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>91</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color: rgb(255, 255, 255) }</string>
    </property>
    <property name="text">
     <string>Muskelaufbau - Negativ</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="weight_label">
    <property name="geometry">
     <rect>
      <x>871</x>
      <y>340</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>60</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color: rgb(193, 254, 255) }</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="kg">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>386</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color: rgb(202, 221, 226) }</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>kg</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="loops_counter">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>150</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Light</family>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color: rgb(82, 91, 110) }</string>
    </property>
    <property name="text">
     <string>x0</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QGraphicsView" name="star">
    <property name="geometry">
     <rect>
      <x>606</x>
      <y>36</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/p1/res/star.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="hardware_controls_label">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>10</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color: rgb(62, 91, 113) }</string>
    </property>
    <property name="text">
     <string>HARDWARE CONTROLS</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="sensor1">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>100</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color: rgb(255, 255, 255) }</string>
    </property>
    <property name="text">
     <string>Sensor 1</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lever_pos_2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>280</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color: rgb(255, 255, 255) }</string>
    </property>
    <property name="text">
     <string>Lever position</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lever_pos">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>320</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color: rgb(255, 255, 255) }</string>
    </property>
    <property name="text">
     <string>50.00%</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGraphicsView" name="scroller_bgn">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>210</y>
      <width>41</width>
      <height>434</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/p1/res/scroller.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGraphicsView" name="scroller_ticks">
    <property name="geometry">
     <rect>
      <x>1318</x>
      <y>203</y>
      <width>117</width>
      <height>448</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/p1/res/stripes.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="clock">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>671</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="training_progress">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>635</y>
      <width>826</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/p1/res/progress.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="training_status">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>635</y>
      <width>195</width>
      <height>130</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(185, 229, 34);
color:rgb(48,53,62);</string>
    </property>
    <property name="text">
     <string>Ready</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="time_spent">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>460</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color: rgb(255, 255, 255) }</string>
    </property>
    <property name="text">
     <string>time</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="last_time">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>490</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color: rgb(255, 255, 255) }</string>
    </property>
    <property name="text">
     <string>last time 0ms</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="target_time">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>520</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color: rgb(255, 255, 255) }</string>
    </property>
    <property name="text">
     <string>over time 0ms</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>g1</zorder>
   <zorder>label</zorder>
   <zorder>name</zorder>
   <zorder>time</zorder>
   <zorder>sek</zorder>
   <zorder>training_title</zorder>
   <zorder>kg</zorder>
   <zorder>loops_counter</zorder>
   <zorder>star</zorder>
   <zorder>hardware_controls_label</zorder>
   <zorder>sensor1</zorder>
   <zorder>lever_pos_2</zorder>
   <zorder>lever_pos</zorder>
   <zorder>scroller_bgn</zorder>
   <zorder>scroller_ticks</zorder>
   <zorder>clock</zorder>
   <zorder>training_progress</zorder>
   <zorder>training_status</zorder>
   <zorder>weight_label</zorder>
   <zorder>time_spent</zorder>
   <zorder>last_time</zorder>
   <zorder>target_time</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1353</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionShow_hardware_Controls"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionLoad_texture_now"/>
    <addaction name="actionRepaint_OpenGL"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDebug"/>
   <addaction name="menu"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionLoad_texture_now">
   <property name="text">
    <string>Load texture now</string>
   </property>
  </action>
  <action name="actionRepaint_OpenGL">
   <property name="text">
    <string>Repaint OpenGL</string>
   </property>
  </action>
  <action name="actionShow_hardware_Controls">
   <property name="text">
    <string>Show Hardware Controls</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
